CorsixTH/Lua/hospital.lua

```diff
@@ -187,12 +186,15 @@ function Hospital:Hospital(world, avail_rooms, name)
     }
   end
 
-  self.policies["staff_allowed_to_move"] = true
-  self.policies["send_home"] = 0.1
-  self.policies["guess_cure"] = 0.9
-  self.policies["stop_procedure"] = 1 -- Note that this is between 1 and 2 ( = 100% - 200%)
-  self.policies["goto_staffroom"] = 0.6
-  self.policies["grant_wage_increase"] = TheApp.config.grant_wage_increase
+  local gbv = level_config.gbv
+  self.policies = {
+    staff_allowed_to_move = true,
+    send_home = 0.1,
+    guess_cure = 0.9,
+    stop_procedure = 1, -- Note that this is between 1 and 2 ( = 100% - 200%)
+    goto_staffroom = gbv.Tired / 1000, -- Defaults to 0.6
```

Comment:
```suggestion
    goto_staffroom = gbv.Tired / 1000, -- Defaults to 0.6 on normal difficulty
```

Reply:
It's the same on all difficulties, looking at easy00.sam and hard00.sam

Reply:
EASY00 lists 700/800/900; FULL00 and HARD00 are 600/700/800